spec/support/page_helpers.rb

```diff
@@ -215,6 +210,16 @@ def preview_teacher_assessment_recommendation_award_page
       PageObjects::AssessorInterface::PreviewTeacherAssessmentRecommendationAward.new
   end
 
+  def professional_standing_request_page
```

Comment:
These two methods should be prefixed with `assessor_`, we didn't do it in the past but it's something we've been doing going forward as the number of pages has increased.